### PR TITLE
SB-8560: add latlon for non-US countries

### PR DESCRIPTION
**Pull request by fhuspokeo**
For SEO Phone Revamp (latlong was taken from simple google search)

### CODE REVIEW

File: lib/data/AREA_CODE_META_INFO.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@
   :country: CANADA
   :service: y
   :description: ''
+  :latlon: '53.7609,-98.8139'
```
Comment (evitero) [inline lib/data/AREA_CODE_META_INFO.yml]: do you think it makes sense to have them as separate values?

Comment (amamtura) [reply lib/data/AREA_CODE_META_INFO.yml]: I like the idea of separate values

**Comment by amamtura** [comment]
> So we don't need the lat/lons for US area codes? Seems like a user of this gem would want it for all of them, no?
> 
> Also, @wtiu-spokeo, does data quality have some way of getting this information/updating it? I'm not sure how often a lat/long will change for an area code...but seems like something we'd want to keep updated

Like both points

**Comment by fhuspokeo** [comment]
Lat/lons for US area codes are not needed because we have finer granularity for their lat/long from their county/city/zip code information. The latlons are needed for these because of desire to display on a map. I can split them into lat, long though that just entails post-processing when I get the coordinates.

**Comment by evitero** [comment]
@fhuspokeo Ah I see, so the client actually wants them in `lat,lon` format? Might make sense to just leave them as is then. 

Normally I'd say standardize to match other stuff in the gem, since this gem is (was?) meant to be client agnostic...but since this is already pretty specific anyway (only lat,lon for non-us) it could be justifiable to go all in lol

**Pull request merged**